common/logisticspipes/utils/gui/InputBar.java

```diff
@@ -21,207 +22,72 @@
 
 	public int minNumber = 0;
 
-	public String input1 = "";
-	public String input2 = "";
-	private boolean isActive;
-	private boolean displaycursor = true;
-	private long oldSystemTime = 0;
-	private int searchWidth;
-	private boolean numberOnly;
-	private Align align;
-
-	private final FontRenderer fontRenderer;
-	private final LogisticsBaseGuiScreen screen;
-	private int left, top, heigth, width;
-
-	public InputBar(FontRenderer fontRenderer, LogisticsBaseGuiScreen screen, int left, int top, int width, int heigth) {
-		this(fontRenderer, screen, left, top, width, heigth, true);
+	// private Align align; // TODO: implement alignment support again
```

Comment:
There is not going to be a new TODO in code. Open an issue if you want to keep track of this. That's what they are for.

Reply:
> you cant hold down the backspace key to delete several characters

`Keyboard.setRepeatEvents(true)`, I believe? And when the gui closes, `Keyboard.setRepeatEvents(false)`.

> when in creative mode and you try to enter a `p` into the text field, the Favourite Potions gui opens. Dunno if thats vanilla (it seems to be) and I dont know if that happened before

That happens in the creative item menu too, and it's annoying. Seems to be a bug in whatever adds that gui.